### PR TITLE
Maintenance: Extend and update docker-compose environment list

### PR DESCRIPTION
**Pull request by MrGeneration**
These changes have been introduced with Zammad 5.1 and seem to have slipped. :)

### CODE REVIEW

File: install/docker-compose/environment.rst [hunk 1]
```diff
@@ -72,12 +72,24 @@ ELASTICSEARCH_SSL_VERIFY: ``true``
 Memcached
 ---------
 
-MEMCACHED_HOST: ``zammad-memcached``
+MEMCACHED_SERVERS: ``zammad-memcached``
```
Comment (mgruner) [inline install/docker-compose/environment.rst:75]: I believe it's `MEMCACHE_SERVERS`, without the `D`.

File: install/docker-compose/environment.rst [hunk 2]
```diff
@@ -72,12 +72,24 @@ ELASTICSEARCH_SSL_VERIFY: ``true``
...
    Provide your own Memcached instance if you already have one existing.
 
+   .. warning:: Was ``MEMCACHED_HOST`` before 5.0.x!
+
 MEMCACHED_PORT: ``11211``
```
Comment (mgruner) [inline install/docker-compose/environment.rst:80]: This should be dropped, it is no longer used.

**Review by mgruner — CHANGES_REQUESTED** [review]
Looks good in general, but some corrections should be applied.

**Comment by MrGeneration** [comment]
Oh noes, you're right. Thank you!

File: install/docker-compose/environment.rst [hunk 3]
```diff
@@ -72,12 +72,24 @@ ELASTICSEARCH_SSL_VERIFY: ``true``
...
 
-MEMCACHED_HOST: ``zammad-memcached``
+MEMCACHE_SERVERS: ``zammad-memcached``
    Provide your own Memcached instance if you already have one existing.
 
-MEMCACHED_PORT: ``11211``
+   .. warning:: Was ``MEMCACHED_HOST`` before 5.0.x!
+
+MEMCACHE_PORT: ``11211``
    Memcacheds default port.
```
Comment (mgruner) [inline install/docker-compose/environment.rst:81]: Sorry, I may not have been clear enough on this. These lines should be removed, that variable does not exist.

**Review by mgruner — CHANGES_REQUESTED** [review]

File: install/docker-compose/environment.rst (hunk 3 above)
Comment (MrGeneration) [reply install/docker-compose/environment.rst]: Ugh dunno where I picked that up sorry.

**Review by mgruner — APPROVED** [review]
Thanks!

**Pull request merged**